### ISSUE TITLE
[Usability] User can't identify an element or its state.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80610]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User can't identify an element or its state.
**VA.gov Experience Standard - category:** Usability
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the Claim Overview (/overview) page, the Show past updates/Hide past updates button doesn't communicate anything about its state to screen reader users. When pressing that button, screen readers don't announce anything. Users may guess correctly that content was expanded, or they may guess incorrectly that the button press didn't occur or that the button is broken.

## Recommended action

DK is already tracking this in #78904, and I agree with his remediation guidance.

## References

- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG21/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jerekshoe** [comment]
This issue has been remediated
